docs/running-on-kubernetes.md

```diff
@@ -44,8 +57,8 @@ are set up as described above:
       --kubernetes-namespace default \
       --conf spark.executor.instances=5 \
       --conf spark.app.name=spark-pi \
-      --conf spark.kubernetes.driver.docker.image=registry-host:5000/spark-driver:latest \
-      --conf spark.kubernetes.executor.docker.image=registry-host:5000/spark-executor:latest \
+      --conf spark.kubernetes.driver.docker.image=kubespark/spark-driver:0.1.0-alpha.1 \
```

Comment:
Good catch! We wanted to call the spark alpha release: `2.1.0-k8s-support-0.1.0-alpha.1`. Do you think it makes sense to tag the docker image the same way? If we have agreement on that, I'll be glad to change this.

We had agreement in an [earlier SIG meeting](https://docs.google.com/document/d/1pnF38NF6N5eM8DlK088XUW85Vms4V2uTsGZvSp8MNIA/edit) that we wanted to avoid having the k8s version and instead have that in the release notes.